translations/messages+intl-icu.de.yml

```diff
@@ -1277,6 +1277,10 @@ error.statement.submit.nowriteaccess: "In dem Verfahren können derzeit keine St
 error.statement.submit.phase: "In der aktuellen Phase des Verfahrens können keine Stellungnahmen eingereicht werden."
 error.statement.update: "Beim Aktualisieren der Stellungnahme ist ein Fehler aufgetreten."
 error.statements.marked.submitted: "Die Stellungnahmen konnten nicht eingereicht werden!"
+error.statements.zip.export.generic.xlsx: "Die xlsx Datei konnte nicht dem Zip hinzugefügt werden."
+error.statements.zip.export.generic.attachment: "Beim Hinzufügen eines Anhang ist ein unbekannter Fehler aufgetreten."
```

Comment:
typo: "eines Anhangs"

Reply:
adjusted in [9f8a2ef](https://github.com/demos-europe/demosplan-core/pull/2337/commits/9f8a2efe24a649667e76190d9d2c369767da6047)